random_menu.rb

```diff
@@ -0,0 +1,108 @@
+### ARRAYS FOR FIRST THROUGH OPTIONAL #2
+# adjs = ["Soft", "Hard", "Crunchy", "Crispy", "Creamy", "Candied", "Flaky", "Infused", "Juicy", "Spicay"]
+#
+# style = ["Pan-seared", "Cured", "Deep-Fried", "Braised", "Glazed", "Marinated", "Pureed", "Reduction", "Sauteed", "Charbroiled"]
+#
+# foods = ["Chicken", "Beef", "Fish", "Clams", "Ribs", "Shrimp", "Scallop", "Vegetables", "Tofu", "Lamb"]
+
+### MAIN PROBLEM USE .SAMPLE METHOD TO PULL RANDOMLY FROM EACH ARRAY
+# puts "Here are your 10 menu items for today!"
+# puts
+# puts "=" * 30
+# puts
+# 10.times do |i|
+#   puts "#{i +1}. #{adjs.sample} #{style.sample} #{foods.sample}"
+# end
+
+### OPTIONAL ENHANCEMENT #1 - Do not repeat menu item names
+# puts
+# puts "Here are your 10 menu items for today!"
+# puts
+# puts "=" * 30
+# puts
+# 10.times do |i|
+#   # Store random value
+#   adjs_value = adjs.sample
+#   style_value = style.sample
+#   foods_value = foods.sample
+#   # Print menu items
+#   puts "#{i + 1}. #{adjs_value} #{style_value} #{foods_value}"
+#
+#   # Delete from array so that it will not be used again
+#   adjs.delete(adjs_value)
+#   style.delete(style_value)
```

Comment:
This is a clever way to make sure all elements are unique. I also like that you've broken out the three steps (choose a word, print the word, delete it from the array) rather than trying to do it all in one line - good style instincts.